examples/satellite-ibm/variables.tf

```diff
@@ -110,7 +110,7 @@ variable "cluster_profile" {
 variable "create_cluster" {
   description = "Create Cluster: Disable this, not to provision cluster"
   type        = bool
-  default     = true
+  default     = false
```

Comment:
i think you changed the wrong one here. The new azure cluster default is still true. This file is ibm vpc.

Reply:
Oops! Fix